docs_src/tutorials/hello_xls.md

```diff
@@ -107,14 +107,14 @@ up! Open a terminal and execute the following in the XLS checkout root
 directory:
 
 ```
-$ ./bazel-bin/xls/dslx/interpreter_main hello_xls.x
+$ ./bazel-bin/xls/dslx/interpreter_main --logtostderr hello_xls.x
 ```
 
 You should see the following output:
 
 ```
 [ RUN UNITTEST  ] hello_test
-trace of hello_string @ hello.x:4:17-4:31: [72, 101, 108, 108, 111, 44, 32, 88, 76, 83, 33]
+... trace of hello_string: [72, 101, 108, 108, 111, 44, 32, 88, 76, 83, 33]
```

Comment:
is the `...` intentional?

Reply:
Yes - I omitted the timestamp, thread ID, filename, and line number for brevity. The complete output looks something like this:

I1128 12:37:01.486118    1057 hello_xls.x:2] trace of hello_string: [72, 101, 108, 108, 111, 44, 32, 88, 76, 83, 33]

I suppose we could spell it all out in general terms:

IMMDD HH:MM:SS.MICROS  THRDID hello_xls.x:2] trace of hello_string: [72, 101, 108, 108, 111, 44, 32, 88, 76, 83, 33]

Do you have a preference?